sound/soc/sof/ipc4-topology.c

```diff
@@ -168,6 +168,75 @@ static void sof_ipc4_dbg_audio_format(struct device *dev,
 	}
 }
 
+static const struct sof_ipc4_audio_format *
```

Comment:
commit message: I have idea what this means

"This patch will set copier sink from the connected pin format."

The connected pin format can only be used to set a sink pin format, no?

Reply:
The already merged PR name the function as `sof_ipc4_set_copier_sink_format` to align with firmware naming, it is actually setting the copier source(output) pin format from the connected sink(input) pin format of the downstream widget.

I will update the commit message to: set copier source pin from connected sink pin format